### ISSUE TITLE
508-defect-3 [COGNITION]: VAOS VA Care - Calendar should announce "date, selected" when users check a radio button and close the picker

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#24634]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

As a screen reader user, I would like to hear a date and AM/PM selected when I check a box in the Community Care calendar date picker. This will help reinforce that I've picked a time slot successfully.

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

The VA care calendar date picker cells have buttons to toggle the time picker. When users select a time slot, we add a visual check to the button. We should update the aria-label to say "Friday, May 4, selected" so it reads that additional information to screen readers. Screenshot and code snippet below.

## Acceptance Criteria

- [ ] Community Care date picker button aria-label data updates correctly when a radio button is checked 

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment/select-date
* Test user Judy Morrison

## Proposed Solution (if known)

```diff
! We should append the string "Selected." when users check a radio button

<button
  aria-controls="vaos-options-container-2021-06-04"
  aria-describedby="vaos-calendar-instructions-5"
  aria-expanded="true"
- aria-label="Friday, June 4th"
+ aria-label="Friday, June 4th. Selected."
  class="vaos-calendar__calendar-day-button"
  id="date-cell-2021-06-04"
  type="button"
>
  <i class="fas fa-check vads-u-color--white vaos-calendar__fa-check-position"></i>
  4
  <span class="vaos-calendar__day--selected-triangle"></span>
</button>
```

## WCAG or Vendor Guidance (optional)

![Screen Shot 2021-05-12 at 4 21 30 PM](https://user-images.githubusercontent.com/934879/118046254-af942d00-b33e-11eb-8905-f999cc2c87c5.png)
